samples/bluetooth/broadcast_audio_source/Kconfig

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 Demant A/S
+# SPDX-License-Identifier: Apache-2.0
+
+
+mainmenu "Bluetooth: Broadcast Audio Source"
+
+choice BT_BAP_LC3_PRESET
+	prompt "The BAP LC3 Preset to be used"
+	default BAP_BROADCAST_16_2_1_preset
+
+config BAP_BROADCAST_16_2_1_preset
+	bool "BT_BAP_LC3_BROADCAST_PRESET_16_2_1 preset"
+	help
+	  Using the BT_BAP_LC3_BROADCAST_PRESET_16_2_1 preset.
+
+config BAP_BROADCAST_24_2_1_preset
```

```suggestion
config BAP_BROADCAST_16_2_1
	bool "BT_BAP_LC3_BROADCAST_PRESET_16_2_1 preset"
	help
	  Using the BT_BAP_LC3_BROADCAST_PRESET_16_2_1 preset.

config BAP_BROADCAST_24_2_1
```

I would probably just skip the `_preset` suffix, especially since it's part of the `choice` as well